### PR TITLE
prow/bugzilla: Switch to 4.5 post GA rules

### PR DESCRIPTION
**Pull request by sdodson**
/cc @eparis 
/hold until switchover

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -322,14 +322,10 @@ bugzilla:
       target_release: 4.4.z
       validate_by_default: true
     openshift-4.5:
-      dependent_bug_states:
```
Comment (eparis) [inline core-services/prow/02_config/_plugins.yaml:325]: Should we land this now? QE should be testing 4.6, right?

Comment (sdodson) [reply core-services/prow/02_config/_plugins.yaml:325]: They are, I'm just verifying or managing that by exception on my own right now.

**Comment by sdodson** [comment]
@petr-muller PTAL

**Pull request merged**